kakao/src/main/kotlin/com/agoda/kakao/edit/TextInputLayoutAssertions.kt

```diff
@@ -34,6 +37,15 @@ interface TextInputLayoutAssertions : BaseAssertions {
         })
     }
 
+    fun hasHint(@StringRes resId: Int) {
+        hasHint(getResourceString(resId))
+    }
+
+    private fun getResourceString(@StringRes resId: Int): String {
+        val targetContext: Context = InstrumentationRegistry.getInstrumentation().targetContext
+        return targetContext.resources.getString(resId)
+    }
+
```

Comment:
`hasHint` should be a part of `KTextView` not a `KTextInputLayout`

Could you move it please to `TextViewAssertions`

Reply:
`TextViewAssertions` already has `fun hasHint(@StringRes resId: Int)` method 🤔

Reason I need in on `TextInputLayoutAssertions` is that I need to verify part of my screen - `KTextInputLayout` where `hint` is set:
![image](https://user-images.githubusercontent.com/530122/83667082-2de43d00-a5ce-11ea-90f9-38983e267527.png)

![image](https://user-images.githubusercontent.com/530122/83667047-1d33c700-a5ce-11ea-9f34-9812e566ee81.png)

Am I missing something?

Reply:
```
 inputLayout {
                edit {
                    hasHint(R.srtings.my_hint)
                }
```

Reply:
I am not sure what is your point here, but current behaviour is broken:

```
meetingNameTextInputLayout {
                isVisible()

                hasHint(R.string.meeting_name) // Error (Adressed by this PR)
                hasHint("ABC") // OK
                
                edit {
                    hasHint(R.string.meeting_name) // OK
                    hasHint("ABC") // OK
                }
            }
```

Am I missing something?